### DISCUSSION TITLE
Opening URLs without modifier keys (just click) triggers a warning

### DISCUSSION THREAD

**Discussion by kdato** [kovidgoyal/kitty#3494] [Q&A]
I don't want to open an issue just yet because this might be user error.

I like clicking on URLs without kitty_mod + mouse. To do that I just set 'open_url_modifiers none' in kitty.conf. It works as I want it to, but I get a warning like this:

1) In a running instance of kitty, I open a new instance
2) The old instance says "[108 12:46:18.623401] Shortcut: none has unknown modifier, ignoring"

How do I get rid of the warning?

**Comment by kovidgoyal** [comment]
It's harmless ignore it.
